elfi/examples/gauss.py

```diff
@@ -22,18 +21,18 @@ def gauss(mu, sigma, n_obs=50, batch_size=1, random_state=None):
 
     Returns
     -------
-    y_obs : array_like
+    x_obs : array_like
```

Comment:
Not a big thing, but I think we as well as most literature use y? 

(The same in many occurrences below...)

Reply:
Okay, you are right on this one. Better set the convention and stick to it, most of literature indeed prefers y over x. Will change these back